Makefile

```diff
@@ -92,12 +92,14 @@ GOCACHE ?= "$(shell echo ${PWD})/out/gocache"
 
 GIT_COMMIT_ID = $(shell git rev-parse --short HEAD)
 
-OPERATOR_VERSION ?= 0.0.10
+OPERATOR_STABLE_VERSION ?= 0.0.11
+OPERATOR_DEV_VERSION ?= $(OPERATOR_STABLE_VERSION)-dev
 OPERATOR_GROUP ?= ${GO_PACKAGE_ORG_NAME}
-OPERATOR_IMAGE ?= quay.io/${OPERATOR_GROUP}/${GO_PACKAGE_REPO_NAME}
-OPERATOR_TAG_SHORT ?= $(OPERATOR_VERSION)
-OPERATOR_TAG_LONG ?= $(OPERATOR_VERSION)-$(GIT_COMMIT_ID)
-
+OPERATOR_APP ?= ${GO_PACKAGE_REPO_NAME}
+OPERATOR_IMAGE_NAME ?= app-binding-operator
```

Comment:
Will we keep this name "app binding" for the long term - as opposed to "service binding operator?"

Reply:
The image repository needs to be different than the name of the application - quay would not allow to push operator package with the name same as the image.